### PR TITLE
[Translation] Add Brazilian portuguese translation (partial)

### LINKED ISSUE #527: Translate to pt-BR (Brazilian Portuguese)
Create Brazilian Portuguese translation (pt-BR).

### PR DESCRIPTION
**Pull request by ThiagoWhispher**
Fix #527 

Translated `potato.js` and `print.js`.

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import es from './es';
...
+import ptBR from './pt-BR';
 export default {
   ...de,
   ...en,
   ...es,
   ...fr,
   ...ja,
-  ...zh
+  ...zh,
+  ...ptBR
```
Comment (yanfali) [inline src/i18n/index.js]: alphabetical ordering please

**Comment by yanfali** [comment]
@gondolindrim @fcoury can you guys verify these translations if you have the time

File: src/i18n/index.js (hunk 1 above)
Comment (ThiagoWhispher) [reply src/i18n/index.js]: Done

**Comment by noroadsleft** [comment]
The translations to `potato.js` and `print.js` look good to my admittedly basic level of Portuguese.

*Obrigado, senhor!*

**Comment by Gondolindrim** [comment]
After the two commits updating index.js, portuguese translations are spot-on.

**Comment by fcoury** [comment]
Second what @Gondolindrim said. Great work!

**Comment by ThiagoWhispher** [comment]
qmk/qmk_firmware#6924

**Comment by ThiagoWhispher** [comment]
> you should check the browser string I think it may be 'pt-BR'
@yanfali 
According to travis, 'pt-BR' is not accepted, should I change to 'ptBR' or have any suggestions?

**Comment by yanfali** [comment]
> > you should check the browser string I think it may be 'pt-BR'
> 
> @yanfali
> According to travis, 'pt-BR' is not accepted, should I change to 'ptBR' or have any suggestions?

Wrap it in single quotes, and it'll work. It just needs to be a string to keep travis happy. e.g. 'pt-BR'

**Comment by yanfali** [comment]
@ThiagoWhispher you'll get credit after we merge. We will take all PRs that qualify. Don't worry.

File: src/i18n/pt-BR/index.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  pt-BR: {
```
Comment (yanfali) [inline src/i18n/pt-BR/index.js]: @ThiagoWhispher this is the one that needs wrapping with single quotes.

Comment (ThiagoWhispher) [reply src/i18n/pt-BR/index.js]: Done @yanfali :D

**Comment by yanfali** [comment]
@ThiagoWhispher one last step, sorry. https://github.com/qmk/qmk_configurator/tree/master/src/i18n do the 5th step and add Brazilian to the list of languages that can be set manually. Many thanks.

**Comment by ThiagoWhispher** [comment]
> @ThiagoWhispher one last step, sorry. https://github.com/qmk/qmk_configurator/tree/master/src/i18n do the 5th step and add Brazilian to the list of languages that can be set manually. Many thanks.

Want me to change 'ptBR' in `index.js` to 'pt-BR'?

**Comment by yanfali** [comment]
@ThiagoWhispher you could, but it's not super important. Brazilian is the first language we've had that we've done that with. For example, I'm waiting for @zvecr to send me a en-Gb any day now, telling me how colonials can't spell.

**Comment by yanfali** [comment]
@ThiagoWhispher I will send you a PR

**Comment by ThiagoWhispher** [comment]
> @ThiagoWhispher I will send you a PR

OK @yanfali. If you wish, I can also change 'br' instead of 'pt-BR'

**Comment by yanfali** [comment]
@ThiagoWhispher https://github.com/ThiagoWhispher/qmk_configurator/pull/1

**Comment by ThiagoWhispher** [comment]
> @ThiagoWhispher [ThiagoWhispher#1](https://github.com/ThiagoWhispher/qmk_configurator/pull/1)

Merged @yanfali

**Comment by yanfali** [comment]
@ThiagoWhispher LGTM (looks good to me). Are you ok with merging?

**Comment by ThiagoWhispher** [comment]
Of course, I will be happy to contribute a merged PR.

**Comment by ThiagoWhispher** [comment]
I also worked on translating qmk/qmk.fm#62, could they look when available? I also want to work on translating the qmk/qmk_firmware repository.

**Comment by yanfali** [comment]
@ThiagoWhispher we're a bit busy right now, but I'm sure someone will get to it eventually

**Comment by yanfali** [comment]
@ThiagoWhispher @Gondolindrim @fcoury thank you all so much for your help and contributions

**Comment by sanxy** [comment]
this is cool

**Comment by ThiagoWhispher** [comment]
@yanfali 
I participated and completed the Hacktoberfest event held by QMK, but have not received any emails about. How is the progress of the delivery of the swags?

**Comment by yanfali** [comment]
@ThiagoWhispher still in progress, sorry. @jackhumbert

**Comment by ThiagoWhispher** [comment]
OK @yanfali :D

**Pull request merged**